composer.json

```diff
@@ -24,7 +24,7 @@
     "post-update-cmd": "if php -r 'exit( version_compare( phpversion(), \"8.1\", \">=\" ) ? 0 : 1 );'; then composer --working-dir=build-cs update --no-interaction; else echo 'Skipping composer update for build-cs since not on PHP 8.1+. You are running: '; php -v; fi",
     "phpstan": "build-cs/vendor/bin/phpstan analyse --memory-limit=2048M -c phpstan.neon.dist",
     "format": "build-cs/vendor/bin/phpcbf --standard=phpcs.xml.dist --report-summary --report-source",
-    "lint": "build-cs/vendor/bin/phpcs --standard=phpcs.xml.dist",
+    "lint": "build-cs/vendor/bin/phpcs",
```

Comment:
The same is updated in https://github.com/WordPress/performance/pull/995. I think a rebase with trunk should update it.

Reply:
Open PR #1007 that merge trunk branch changes.